### PR TITLE
HOT-341-int-pointer-helpers

### PR DESCRIPTION
**Pull request by sebastian-bojara**

### CODE REVIEW

File: pointers.go [hunk 1]
```diff
@@ -14,3 +14,11 @@ func PtrString(s *string) string {
 	}
 	return *s
 }
+
+func IntPtr(i int) *int {
+	return &i
+}
+
+func PtrInt(i *int) int {
+	return *i
```
Comment (kglusinski) [inline pointers.go:23]: actually i can be nil and in this case you will get nil pointer dereference error

**Review by kglusinski — CHANGES_REQUESTED** [review]

File: pointers.go (hunk 1 above)
Comment (kglusinski) [reply pointers.go:23]: I would remove this function. You can  easily do `*variable` to dereference it and handle error. Much easier than create pointer from some literal like `5`

Comment (sebastian-bojara) [reply pointers.go:23]: it can be beneficial for example for tests where you created structures, so you have many literals

Comment (kglusinski) [reply pointers.go:23]: Anyway it will cause error in case of nil. So at least return `(int, err)` if `i == nil`

**Review by kglusinski — APPROVED** [review]

**Pull request merged**